audio-processing/wav2vec2-inference-checkpoint.ipynb

```diff
@@ -2,8 +2,6 @@
  "cells": [
```

Comment:
<p>Remove and move instructions for running on Paperspace up:</p><p> </p><ul><li>A Poplar SDK environment enabled (see the&nbsp;<a href="https://docs.graphcore.ai/en/latest/getting-started.html" target="_blank">Getting Started</a>&nbsp;guide for your IPU system)</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/graphcore/Graphcore-HuggingFace-fork/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>